### DISCUSSION TITLE
Are real scenes released with the dataset?

### DISCUSSION THREAD

**Discussion by lunayach-wai** [allenai/ai2thor#720] [Q&A]
Sorry, this might be a stupid question but The [paper](https://arxiv.org/pdf/2004.06799.pdf) mentions that "ROBOTHOR consists of a set of 89 apartments, 75 in train/Val (we use 60 for training and 15 for validation), **4 in test-dev (which are used for validation in the real world) and 10 in test-standard (blind physical test set) drawn from a set of 15, 2 and 5 wall layouts respectively**". 

But in the API [documentation](https://ai2thor.allenai.org/robothor/documentation/#initialization-scene) for the scene, it only talks about Train and Val scenes. I was interested in leverage the sim2real mapping in scenes, but are real scenes not released?

**Comment by roozbehm** [comment]
Unfortunately, due to COVID, we had to shut down the lab used for RoboTHOR. We hope to resume the real part of RoboTHOR soon when our researchers and engineers go back to the office.
